opendatahub-io/data-science-pipelines-operator
# feat: Removed traces of argo server from default deployment.

## The issue resolved by this Pull Request:
Resolves [RHOAIENG-3895](https://issues.redhat.com/browse/RHOAIENG-3895)

## Description of your changes:
Removed the deployment files for argo server as it isn't required by default and it was the team's concensus as well in [RHOAIENG-2873](https://issues.redhat.com/browse/RHOAIENG-2873)



## Testing instructions
1. Deploy DSPO using `make deployODH IMG="quay.io/opendatahub/data-science-pipelines-operator:pr-587"`
2. Deploy any sample DSPA. Example - `/config/samples/v2/dspa-simple`
3. Run Pipelines and ensure everything works like it normally does.

**NOTE**
If you deploy DSPO using `make deploy IMG="...`, ensure to also do `make argodeploy` as these will install all the CRDs,RBACs and whatnot required for argo workflow controller to work



## Checklist
